noir-projects/noir-protocol-circuits/.gitignore

```diff
@@ -2,4 +2,5 @@ Verifier.toml
 target
 crates/autogenerated
 /Nargo.toml
-/private_kernel_reset_dimensions.json
\ No newline at end of file
+/private_kernel_reset_dimensions.json
+install-state.gz
```

Comment:
Added this to gitignore as it was apparently accidentally committed. See [this discussion](https://aztecprotocol.slack.com/archives/C02M7VC7TN0/p1734734261772969)